frontend/next-env.d.ts

```diff
@@ -1,2 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
```

Comment:
Is this from the next 11 upgrade or separate? Looks like it's for importing images, does it come from https://nextjs.org/blog/next-10#built-in-image-component-and-automatic-image-optimization ?

Reply:
Yeap, this comes from the Next 11 upgrade! I'm honestly not quite sure what this file does, but it is autogenerated by Next.